src/components/ContactComponent/ContactComponent.scss

```diff
@@ -0,0 +1,110 @@
+@import "../../styles/utils/variables";
+@import "../../styles/utils/mixins";
+
+.contact {
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+
+  .contact__details {
+    @include linear-gradient($cc-purple, $cc-purple-dark);
+    width: 250px;
+    height: 600px;
+    padding: 10px;
+    color: $cc-white;
+
+    p {
+      font-size: 1.5rem;
+      margin-left: 20px;
+    }
+
+    .wrapper {
+      margin-left: 10px;
+
+      .facebook, .github, .site {
+        margin-top: 50px;
+        margin-left: 30px;
+        font-size: 3rem;
+      }
+    }
+  }
+
+  .contact__form {
+    width: 600px;
+    height: 500px;
+    margin: auto -120px;
+    border: 1px solid black;
+    background-color: $cc-white;
+    color: $cc-white;
+    font-size: 1.5rem;
+    box-shadow: 0 0 20px $cc-purple-dark;
+    border-radius: 5px;
+
+    p {
+      font-size: 1.5rem;
+      margin-left: 20px;
+    }
+  }
+}
+
+@media screen and (max-width: 883px) {
+  .contact {
+    display: block;
+    margin: auto auto 20px;
```

Comment:
would `margin-bottom: 20px` give us same result? 
we didn't set any `margin` to `.contact` element earlier, so we don't need to override them and `auto` values are default ones for the `margin` property.

Reply:
Unfortunately not. `margin: auto auto 20px` is required so component looks as it should.